modules/copy-post.php

```diff
@@ -0,0 +1,190 @@
+     * @param int       $target_post_id Target post ID.
+     * @return int|bool Meta ID if the key didn't exist, true on successful update, false on failure.
+     */
+    protected function update_featured_image( $source_post, $target_post_id ) {
+        $featured_image_id = get_post_thumbnail_id( $source_post );
+        return update_post_meta( $target_post_id, '_thumbnail_id', $featured_image_id );
+    }
+
+    /**
+     * Update the target post's post format.
+     *
+     * @param WP_Post               $source_post Post object to be copied.
+     * @param int                   $target_post_id Target post ID.
+     * @return array|WP_Error|false WP_Error on error, array of affected term IDs on success.
+     */
+    protected function update_post_format( $source_post, $target_post_id ) {
+        $post_format = get_post_format( $source_post );
+        return set_post_format( $target_post_id, $post_format );
+    }
+
+    /**
+     * Update the target post's title.
+     *
+     * @param string  $post_title Post title determined by `get_default_post_to_edit()`.
+     * @param WP_Post $post       Post object of newly-inserted post.
+     * @return string             Updated post title from source post.
+     */
+    function filter_title( $post_title, $post ) {
+        return $post->post_title;
+    }
+
+    /**
+     * Update the target post's content (`post_content`).
+     *
+     * @param string  $post_content Post content determined by `get_default_post_to_edit()`.
+     * @param WP_Post $post         Post object of newly-inserted post.
+     * @return string               Updated post content from source post.
+     */
+    function filter_content( $post_content, $post ) {
+        return $post->post_content;
+    }
+
+    /**
+     * Update the target post's excerpt.
+     *
+     * @param string  $post_excerpt Post excerpt determined by `get_default_post_to_edit()`.
+     * @param WP_Post $post         Post object of newly-inserted post.
+     * @return string               Updated post excerpt from source post.
+     */
+    function filter_excerpt( $post_excerpt, $post ) {
+        return $post->post_excerpt;
+    }
+
+    /**
+     * Add a "Copy" row action to posts/pages/CPTs on list views.
+     *
+     * @param array   $actions Existing actions.
+     * @param WP_Post $post    Post object of current post in list.
+     * @return array           Array of updated row actions.
+     */
+    function add_row_action( $actions, $post ) {
+        $edit_url = add_query_arg( array(
+            'post_type' => $post->post_type,
+            'copy' => $post->ID,
+            '_wpnonce' => wp_create_nonce( 'jetpack-copy-post' ),
+        ), admin_url( 'post-new.php' ) );
+        $edit_action = array(
+            'copy' => sprintf(
+                '<a href="%s" aria-label="%s">%s</a>',
+                esc_url( $edit_url ),
+                esc_attr( __( 'Copy this post.' ) ),
```

Comment:
Instead of `esc_attr( __(`, you could use `esc_attr__(`:

```suggestion
                esc_attr__( 'Copy this post.', 'jetpack' ) ),
```